plugins/@grouparoo/iterable/src/lib/export/exportProfile.ts

```diff
@@ -0,0 +1,77 @@
+import { ExportProfilePluginMethod } from "@grouparoo/core";
+import { connect } from "../connect";
+import { addToList, removeFromList } from "./listMethods";
+
+export const exportProfile: ExportProfilePluginMethod = async ({
+  appGuid,
+  appOptions,
+  export: {
+    toDelete,
+    newProfileProperties,
+    oldProfileProperties,
+    newGroups,
+    oldGroups,
+  },
+}) => {
+  // if we received no mapped data... just exit
+  if (Object.keys(newProfileProperties).length === 0) {
+    return { success: true };
+  }
+  const client = await connect(appOptions);
+
+  const email = newProfileProperties["email"]; // this is how we will identify profiles
+  if (!email) {
+    throw new Error(`newProfileProperties[email] is a required mapping`);
+  }
+
+  try {
+    let user;
+    try {
+      user = await client.users.get({ email });
```

Comment:
I thin it's possible you won't have to do this because it uses the email to upsert and add/remove from lists.